### PR TITLE
Added Swedish Translation

### PR DESCRIPTION
**Pull request by bittin**

### CODE REVIEW

**Review by beanpole135 — CHANGES_REQUESTED** [review]
All of these translations are in the wrong spot and will not be used they way they are now.
The `type="unfinished"` tag needs to be removed, and the translated text put between the tags like this:
`<translation>This is translated text</translation>`

File: src-qt5/desktop-utils/lumina-photo/i18n/l-photo_sv.ts [hunk 1]
```diff
@@ -6,205 +6,205 @@
     <message>
         <location filename="../mainUI.ui" line="14"/>
         <source>Image Viewer</source>
-        <translation type="unfinished"></translation>
+        <<translation>Bildvisare></translation>
```
Comment (beanpole135) [inline src-qt5/desktop-utils/lumina-photo/i18n/l-photo_sv.ts:9]:
```suggestion
        <translation>Bildvisare</translation>
```

File: src-qt5/desktop-utils/lumina-photo/i18n/l-photo_sv.ts [hunk 2]
```diff
@@ -6,205 +6,205 @@
...
     </message>
     <message>
         <location filename="../mainUI.ui" line="34"/>
         <source>&amp;File</source>
-        <translation type="unfinished"></translation>
+        <<translation>&amp; Fil"></translation>
```
Comment (beanpole135) [inline src-qt5/desktop-utils/lumina-photo/i18n/l-photo_sv.ts:14]:
```suggestion
        <translation>&amp; Fil</translation>
```

**Review by beanpole135 — CHANGES_REQUESTED** [review]
Looks like you got some more stray `<` and `">` in here on the translation lines.
I suggested changes for the first couple to get you started, but you will need to fix up the rest too.

**Comment by bittin** [comment]
fixed

**Review by beanpole135 — APPROVED** [review]
Looks good!
Thanks for getting that cleaned up!

**Pull request merged**